release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1280,14 +1280,13 @@ As part of the continued deprecation of the Operator package manifest format, th
 
-* Previously, if you deleted a master machine using `oc delete machine` in an AWS cluster, the machine was removed from the load balancers. As a result, the load balancer continued to serve requests to the removed control plane machine. With this fix, when you remove a control plane machine, the load balancer no longer serves requests to the machine.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1880757[*BZ#1880757*])
+* Previously, if you deleted a master machine using `oc delete machine` in an AWS cluster, the machine was not removed from the load balancers. As a result, the load balancer continued to serve requests to the removed master machine. With this fix, when you remove a master machine, the load balancer no longer serves requests to the machine. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880757[*BZ#1880757*])
 
-* Previously, when deleting an unreachable machine, the vSphere Virtual Machine Disk (VMDK) that is created for persistent volumes and attached to the node was incorrectly deleted. As a result, the data on the VMDK was unrecoverable. With this release, the vSphere cloud provider checks and detaches these disks from the node if the kubelet is not reachable. With this fix, you can delete an unreachable machine without losing the VMDK. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883993[*BZ#1883993*])
+* Previously, when deleting an unreachable machine, the vSphere Virtual Machine Disk (VMDK) that is created for persistent volumes and attached to the node was incorrectly deleted. As a result, the data on the VMDK was unrecoverable. With this fix, the vSphere cloud provider checks for and detaches these disks from the node if the kubelet is not reachable. As a result, you can delete an unreachable machine without losing the VMDK. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883993[*BZ#1883993*])
 
-* Previously, some newer Amazon Web Services (AWS) instance types were not able to scale from zero when using the Cluster Autoscaler and MachineSets with zero replicas. Because a generated list of AWS instance types was out of date, the list of AWS instance types was updated to include newer instance types. With this fix, more instance types are available to the Cluster Autoscaler for scaling from zero replicas. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896321[*BZ#1896321*])
+* Previously, because a generated list of AWS instance types was out of date, some newer Amazon Web Services (AWS) instance types were not able to scale from zero when using the Cluster Autoscaler Operator and machine sets with zero replicas. The list of AWS instance types was updated to include newer instance types. With this fix, more instance types are available to the Cluster Autoscaler Operator for scaling from zero replicas. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896321[*BZ#1896321*])
```

Comment:
```suggestion
* Previously, because a generated list of AWS instance types was out of date, some newer Amazon Web Services (AWS) instance types were not able to scale from zero when using the Cluster Autoscaler Operator and machine sets with zero replicas. The list of AWS instance types is now updated to include newer instance types. With this fix, more instance types are available to the Cluster Autoscaler Operator for scaling from zero replicas. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896321[*BZ#1896321*])
```